website/docs/d/config_aggregator_configurations.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_config_aggregator_configurations"
+description: |-
+  Get information about config_aggregator_configurations
+subcategory: "Configuration Aggregator"
```

Comment:
Add this subcategory `Configuration Aggregator` in allowed-subcategories.txt file as well